src/core_plugins/tagcloud/public/tag_cloud_vis_params.html

```diff
@@ -11,6 +11,32 @@
     <div class="form-group">
       <label>Font Size</label>
       <div class="tag-cloud-fontsize-slider"></div>
+      <div class="tag-cloud-fontsize-inputs">
+        <div class="tag-cloud-fontsize-input">
+          <input
+            type="number"
+            min="1"
+            max="{{vis.params.maxFontSize}}"
+            class="form-control tag-cloud-fontsize-inputfield"
+            ng-model="vis.params.minFontSize"
+            aria-label="Minimum tag font size"
+            aria-describedby="tagCloudFontSliderMinUnit"
+          >
+          <span id="tagCloudFontSliderMinUnit">px</span>
+        </div>
+        <div class="tag-cloud-fontsize-input">
+          <input
+            type="number"
+            min="{{vis.params.minFontSize}}"
+            max="100"
+            class="form-control tag-cloud-fontsize-inputfield"
+            ng-model="vis.params.maxFontSize"
+            aria-label="Maximum tag font size"
+            aria-describedby="tagCloudFontSliderMaxUnit"
+          >
+          <span id="tagCloudFontSliderMaxUnit">px</span>
```

Comment:
Screen readers read this as "P, X". Should we add `aria-label="pixels"` to make it clearer?